easybuild/easyconfigs/m/Molpro/Molpro-mpp-2010.1.23-Linux_x86_64.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Molpro'
+easyblock = 'EB_Molpro'
+versionprefix = 'mpp-'
+version = '2010.1.23'
+versionsuffix = '-Linux_x86_64'
+
+homepage = 'https://molpro.net'
+description = """Molpro is a complete system of ab initio programs for molecular electronic structure calculations."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+#toolchain = {'name': 'intel', 'version': '2015a'}
+#toolchainopts = {'usempi': True}
+
+# no source URL available, requires registration to download
+sources = ['%(namelower)s-%(versionprefix)s%(version)s%(versionsuffix)s.sh']
+patches = ['%(namelower)s-%(versionprefix)s%(version)s%(versionsuffix)s.patch']
```

Comment:
@valtandor: you explained in #1945 what's going on with the binary patch, but I'm not very comfortable with it

the patch looks like this (GitHub doesn't even support showing it partially), without the binary part:

```diff
--- Molpro/molpro-mpp-2010.1.23-Linux_x86_64.sh	2015-09-02 12:57:08.239724674 +1200
+++ Molpro/molpro-mpp-2010.1.23-Linux_x86_64.sh	2015-09-02 13:01:18.998587159 +1200
@@ -162,7 +162,8 @@
  echo Token sucessfully downloaded and installed
  exit 0
 fi
-molpro_error "Token downloaded unsucessful"
+echo "Token downloaded unsucessful"
+exit 0
 _EOF_
```

Can the patch be done a little bit differently to avoid hitting the binary bit, e.g. by fixing/changing the condition in the `if` above?

Why is this needed b.t.w., what's going on here exactly?